src/TBuffer.h

```diff
@@ -95,7 +95,9 @@ struct TMxpElement
 
 class TBuffer
 {
-    static const QMap<QString, QVector<QChar>> csmEncodingTable; // private!
```

Comment:
Not entirely sure what you are saying here about the disambiguation text unless you mean that [QTBUG-43540](https://bugreports.qt.io/browse/QTBUG-43540) is also factor here.  Which is a pity, as it means ensuring that special marked C/C++ code comments are needed:

* `/*:`string`*/` or `//:`string to replace the "disambiguation" string as a second argument to`tr`
* `//=<`id`>` to provide an "id" based unique identifier string for this instance of this `QString` for id based translators (but probably NOT Qt Liguist!)
* `//~<`field`> <`value`>` to provide an XML marked up `<extra-`field`>value</extra-`field`>` additional extra comment that is included in the `.ts` file (but, as part of the above bug) totally hidden/ignored by Qt Linguist

Of course those comments could get detached or mangled because they are not "part" of the `tr(`...`)` call but that is the Qt people's decision and the rest of us just have to live with it... 😒